### PR TITLE
Added common issues to README

### PR DESCRIPTION
**Pull request by DavidSouthgate**

### CODE REVIEW

**Comment by Mnpn03** [comment]
Would this fit better under “Building” on [this wiki page](https://github.com/tbodt/ish/wiki/Troubleshooting)?

**Comment by tbodt** [comment]
This pull request includes commits apparently intended for another pull request, so i can't merge it.

**Comment by DavidSouthgate** [comment]
Apologies, I thought I had fixed removing that commit. It is sorted now

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ You'll need these things to build the project:
  - Ninja
  - Node and NPM (only when building for iOS)
  - Meson (`pip install meson`)
+ - Parcel (`npm install -g parcel-bundler` - only when building for iOS)
```
Comment (tbodt) [inline README.md]: Installing parcel-bundler manually should not be required.

**Comment by tbodt** [comment]
Closing due to inactivity

**Pull request closed without merging**